test/suites/basic.sh

```diff
@@ -218,7 +218,7 @@ test_instances_config() {
 
   # Setup a MicroCloud with 3 systems, ZFS storage, and a FAN network.
   addr=$(lxc ls micro01 -f csv -c4 | grep enp5s0 | cut -d' ' -f1)
-  lxc exec micro01 --env TEST_CONSOLE=0 -- microcloud init --preseed << EOF
+  lxc exec micro01 --env TEST_CONSOLE=0 -- microcloud init --preseed --lookup-timeout 10 << EOF
```

Comment:
I think I'd rather have the default value bumped to 10s (from the 5s default).

Reply:
Up until this case, we have never seen an issue with finding systems so I felt it might be overkill to force every deployment using `--auto` or `--preseed` to sleep for 10s when 5s appears to be good enough for the majority of cases.

Reply:
Ah sorry I misinterpreted that as being a regular timeout, not an actual wait time. Ignore my point then.